Patterns/Arabic/Arabic-Numbers.yaml

```diff
@@ -639,4 +645,7 @@ RelativeReferenceRelativeToMap: !dictionary
     الذي قبلا الأخير: end
     الثانية الى الاخير: end
     السابق: current    
-...
\ No newline at end of file
+...
+
+
```

Comment:
Minor: Please leave only one blank line at the end of the file

Reply:
@tellarin it was for our Neudesic branch. He (@MuhammedSalamneu) raised to wrong target by mistake.
@MuhammedSalamneu please close this PR.